dev/TEAL.md

```diff
@@ -313,11 +334,12 @@ Global fields are fields that are common to all the transactions in the group. I
 | 7 | LatestTimestamp | uint64 | Last confirmed block UNIX timestamp. Fails if negative. LogicSigVersion >= 2. |
 | 8 | CurrentApplicationID | uint64 | ID of current application executing. Fails if no such application is executing. LogicSigVersion >= 2. |
 | 9 | CreatorAddress | []byte | Address of the creator of the current application. Fails if no such application is executing. LogicSigVersion >= 3. |
+| 10 | CurrentApplicationAddress | []byte | Address that the current application controls. Fails if no such application is executing. LogicSigVersion >= 5. |
```

Comment:
An app always has control of hash('appId' || appId).  I suppose that someday it might be rekeyed, so it would not really have authority.  But this opcode would still return the address.